website/project/views/drafts.py

```diff
@@ -0,0 +1,242 @@
+from flask import request, redirect
+import httplib as http
+from dateutil.parser import parse as parse_date
+
+from modularodm import Q
+from modularodm.exceptions import ValidationValueError
+
+from framework import status
+from framework.mongo.utils import get_or_http_error
+from framework.exceptions import HTTPError
+from framework.status import push_status_message
+from framework.auth.decorators import must_be_logged_in
+
+from website.util.permissions import ADMIN
+from website.project.decorators import (
+    must_be_valid_project,
+    must_have_permission,
+    http_error_if_disk_saving_mode
+)
+from website import settings
+from website.mails import Mail, send_mail
+from website.project import utils as project_utils
+from website.project.model import MetaSchema, DraftRegistration
+from website.project.metadata.utils import serialize_meta_schema, serialize_draft_registration
+from website.project.utils import serialize_node
+
+get_draft_or_fail = lambda pk: get_or_http_error(DraftRegistration, pk)
+get_schema_or_fail = lambda query: get_or_http_error(MetaSchema, query)
+
+@must_have_permission(ADMIN)
+@must_be_valid_project
+def submit_draft_for_review(auth, node, draft_pk, *args, **kwargs):
+    user = auth.user
+
+    draft = get_draft_or_fail(draft_pk)
+    draft.is_pending_review = True
+    draft.save()
+
+    REVIEW_EMAIL = Mail(tpl_prefix='prereg_review', subject='New Prereg Prize registration ready for review')
+    send_mail(draft.initiator.email, REVIEW_EMAIL, user=user, src=node)
+
+    ret = project_utils.serialize_node(node, auth)
+    ret['success'] = True
+    return ret
+
+@must_have_permission(ADMIN)
+@must_be_valid_project
+def draft_before_register_page(auth, node, draft_id, *args, **kwargs):
+    ret = serialize_node(node, auth, primary=True)
+
+    draft = get_draft_or_fail(draft_id)
+    ret['draft'] = serialize_draft_registration(draft, auth)
+    return ret
+
+@must_have_permission(ADMIN)
+@must_be_valid_project
+def draft_before_register(auth, node, draft_id, *args, **kwargs):
```

Comment:
@HarryRybacki one is an API route, while the other is a OsfWebRenderer route. 

If the view code remains the same then this will be consolidated in a helper fn